doc/contributing/coding-conventions.chapter.md

```diff
@@ -1,693 +1,63 @@
 # Coding conventions {#chap-conventions}
 
-## Syntax {#sec-syntax}
-
-- Use 2 spaces of indentation per indentation level in Nix expressions, 4 spaces in shell scripts.
-
-- Do not use tab characters, i.e. configure your editor to use soft tabs. For instance, use `(setq-default indent-tabs-mode nil)` in Emacs. Everybody has different tab settings so it’s asking for trouble.
-
-- Use `lowerCamelCase` for variable names, not `UpperCamelCase`. Note, this rule does not apply to package attribute names, which instead follow the rules in [](#sec-package-naming).
-
-- Function calls with attribute set arguments are written as
-
-  ```nix
-  foo {
-    arg = ...;
-  }
-  ```
-
-  not
-
-  ```nix
-  foo
-  {
-    arg = ...;
-  }
-  ```
-
-  Also fine is
-
-  ```nix
-  foo { arg = ...; }
-  ```
-
-  if it's a short call.
-
-- In attribute sets or lists that span multiple lines, the attribute names or list elements should be aligned:
-
-  ```nix
-  # A long list.
-  list = [
-    elem1
-    elem2
-    elem3
-  ];
-
-  # A long attribute set.
-  attrs = {
-    attr1 = short_expr;
-    attr2 =
-      if true then big_expr else big_expr;
-  };
-
-  # Combined
-  listOfAttrs = [
-    {
-      attr1 = 3;
-      attr2 = "fff";
-    }
-    {
-      attr1 = 5;
-      attr2 = "ggg";
-    }
-  ];
-  ```
-
-- Short lists or attribute sets can be written on one line:
-
-  ```nix
-  # A short list.
-  list = [ elem1 elem2 elem3 ];
-
-  # A short set.
-  attrs = { x = 1280; y = 1024; };
-  ```
-
-- Breaking in the middle of a function argument can give hard-to-read code, like
-
-  ```nix
-  someFunction { x = 1280;
-    y = 1024; } otherArg
-    yetAnotherArg
-  ```
-
-  (especially if the argument is very large, spanning multiple lines).
-
-  Better:
-
-  ```nix
-  someFunction
-    { x = 1280; y = 1024; }
-    otherArg
-    yetAnotherArg
-  ```
-
-  or
-
-  ```nix
-  let res = { x = 1280; y = 1024; };
-  in someFunction res otherArg yetAnotherArg
-  ```
-
-- The bodies of functions, asserts, and withs are not indented to prevent a lot of superfluous indentation levels, i.e.
-
-  ```nix
-  { arg1, arg2 }:
-  assert system == "i686-linux";
-  stdenv.mkDerivation { ...
-  ```
-
-  not
-
-  ```nix
-  { arg1, arg2 }:
-    assert system == "i686-linux";
-      stdenv.mkDerivation { ...
-  ```
+This section has been moved to [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).
```

Comment:
These types of links, while useful when browsing through a browser, now hardcode the branch. I think you can just use relative links.

Reply:
We can't use relative links because this is the rendered manual to be shown on `nixos.org` where the source isn't available.